app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = current_user
```

Comment:
- [ ] Great work on adding this controller file. Follow the previous reviewer's comment on how exactly you have to group the events i.e. all events created by the user, past events attended by the user, and upcoming events the user is going to attend. [Use this link](https://github.com/M-Alamgir/private-events/pull/2/files#r682681194) for more guiding instructions.